### DISCUSSION TITLE
Primera entrega

### DISCUSSION THREAD

**Discussion by moralesluis26** [rramosp/ai4eng.v1#159] [00 - General]
Saludos profesor.
Organizando mi repositorio, me doy cuenta que puedo subir los archivos directamente desde `Add file / Upload files`, mi pregunta es si está bien hacerlo así o si es necesario hacerlo de la forma en la que nos explicó. Le agradezco su atención de antemano.

**Comment by rramosp** [comment]
hola luis david ... en realidad da igual ... lo importante es que estén los
archivos

On Mon, Jul 4, 2022 at 1:28 PM Luis David M ***@***.***>
wrote:

> Saludos profesor.
> Organizando mi repositorio, me doy cuenta que puedo subir los archivos
> directamente desde Add file / Upload files, mi pregunta es si está bien
> hacerlo así o si es necesario hacerlo de la forma en la que nos explicó. Le
> agradezco su atención de antemano.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/159>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIPFOKKY6BZWX7I5VWLVSMUNFANCNFSM52T5TA4A>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
